### PR TITLE
Remove conversion attribution frequency capping - Production

### PR DESCRIPTION
**Pull request by tmancey**
Uplift of https://github.com/brave/brave-variations/pull/1047 and https://github.com/brave/brave-variations/pull/1053

### CODE REVIEW

**Review by iambrianfung — APPROVED** [review]

**Review by iambrianfung — DISMISSED** [review]

**Review by aseren — APPROVED** [review]

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers. QA has verified the PR on `staging` as per https://github.com/brave/brave-variations/issues/1049#issuecomment-2127252824.

**Pull request merged**
